src/service.ts

```diff
@@ -2,40 +2,66 @@ process.env.FORCE_COLOR = "0";
 
 import LRU from "nanolru";
 import path from "path";
-import prettier from "prettier";
-import { Options, format } from "prettier";
+import type { Options } from "prettier";
 
 const configCache = new LRU({ max: 20, maxAge: 60000 });
 
-async function resolveConfig(cwd: string, filepath: string): Promise<Options> {
+async function resolveConfig(
+  prettier: any,
```

Comment:
Can we use a custom interface here? It can have just the resolveConfig function.

Reply:
Sorry, I could have used typeof prettier, late night commit and I was wondering why it was not working as a namespace.